### ISSUE TITLE
Nombre d'éléments à afficher

### ISSUE THREAD

**Issue by alaindebecker** [newmips/newmips#43]
Le nombre d'éléments à afficher est à choisir dans ine liste déroulante femée (50, 200, 500, Tous).
Serait-il possible d'en faire une liste déroulante ouverte? Je dois afficher 3 ou 4 éléments pour un copie d'écran.

**Comment by newmips** [comment]
Non, mais il est possible de configurer le "lengthMenu" du jquery.dataTable.js (aux alentours de la ligne 7940) pour modifier ces valeurs.

**Comment by alaindebecker** [comment]
Merci. Mais coment faire pour ceux qui ne connaissent pas jquery, ou plus simplement ceux qui se sentent découragé à l'idée d'éditer plus de 7940 lignes de code. 

Après tout, s'ils ont choisi newMips, c'est en premier lieu pour ne pas avoir à entrer dans du code.

**Comment by Syno0** [comment]
Il est préférable de ne pas modifier le jQuery mais plutôt la configuration de nos tables.
Pour cela il est possible d'ajouter des choix de listing dans dataTableBuilder.js et simpleTable.js:

**En 2.7:**  /public/js/Newmips/
**Version précédente:** /public/js/plugins/datatables/

à cet endroit il est possible d'ajouter des choix du nombre de ligne à afficher (environ ligne 445): 

![image](https://user-images.githubusercontent.com/14804727/49083080-d5e7b000-f24b-11e8-8b24-466e47ba684f.png)

**Comment by alaindebecker** [comment]
Je pensais plus simplement à changer les `<select>` par des `<datalist>`.

![image](https://user-images.githubusercontent.com/2980348/49087371-11d44280-f257-11e8-9eab-86c387a54afb.png)
